site/docs/releases.md

```diff
@@ -56,7 +56,38 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
   ...
 </dependencies>
 ```
-## 0.12.0 Release Notes
+
+## 0.12.1 Release Notes
+
+Apache Iceberg 0.12.1 was released on November 8th, 2021.
+
+Note that the spark3-runtime jar supports Spark 3.0 and 3.1. Spark 3.2 support will come in the next release or can be tried out with the nightly SNAPSHOT release built from master.
+
+* Git tag: [0.12.1](https://github.com/apache/iceberg/releases/tag/apache-iceberg-0.12.1)
+* [0.12.1 source tar.gz](https://www.apache.org/dyn/closer.cgi/iceberg/apache-iceberg-0.12.1/apache-iceberg-0.12.1.tar.gz) -- [signature](https://downloads.apache.org/iceberg/apache-iceberg-0.12.1/apache-iceberg-0.12.1.tar.gz.asc) -- [sha512](https://downloads.apache.org/iceberg/apache-iceberg-0.12.1/apache-iceberg-0.12.1.tar.gz.sha512)
+* [0.12.1 Spark 3.x runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark3-runtime/0.12.1/iceberg-spark3-runtime-0.12.1.jar)
+* [0.12.1 Spark 2.4 runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime/0.11.1/iceberg-spark-runtime-0.12.1.jar)
+* [0.12.1 Flink runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime/0.12.1/iceberg-flink-runtime-0.12.1.jar)
+* [0.12.1 Hive runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-hive-runtime/0.12.1/iceberg-hive-runtime-0.12.1.jar)
+
+Important bug fixes and changes:
+
+* [\#3135](https://github.com/apache/iceberg/pull/3135) fixes equality-deletes using DATE/TIMESTAMP/TIME types.
+* [\#3071](https://github.com/apache/iceberg/pull/3071) adds predicate push down for checking referenced data files that match row-level updates. Avoids opening manifest files that belong to unrelated partitions when writing row deltas.
+* [\#3078](https://github.com/apache/iceberg/pull/3078) prevents the JDBC catalog from overwriting the `jdbc.user` property if any property called user exists in the environment.
+* [\#3035](https://github.com/apache/iceberg/pull/3035) fixes drop namespace calls with the DyanmoDB catalog.
+* [\#3195](https://github.com/apache/iceberg/pull/3195) allows validating that commits that produce row-level deltas don't conflict with concurrently files. Ensures users can maintain serializable isolation for update/delete operations, including merge operations.
+* [\#3199](https://github.com/apache/iceberg/pull/3199) allows validating that commits that overwrite files don't conflict with concurrently files. Ensures users can maintain serializable isolation for overwrite operations.
+* [\#3264](https://github.com/apache/iceberg/pull/3258) fixes validation exceptions when writing Flink CDC streams to Iceberg tables.
+* [\#3264](https://github.com/apache/iceberg/pull/3264) fixes reading projected map columns from Parquet files written before Parquet 1.11.1. Users can read and project on map columns from Parquet files written with any Parquet version.
```

Comment:
Nit:  Second sentence here is somewhat redundant.  For brevity, we might just want to remove it.

Reply:
Removed.

Reply:
Listed second after Flink CDC.